content/courses/advanced-ada/chapters/limited_types.rst

```diff
@@ -897,47 +897,159 @@ of the task's entries, because that would deadlock. That is, the entry
 call would wait until the task reaches an accept statement, which will
 never happen, because the task will never be activated.
 
-Other usages of extended return statements
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Initialization and function return
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As mentioned in the previous section, the object of limited type returned by
+the initialization function is created *in place*. In other words, the return
```

Comment:
"created" probably should be "built" because the object is not created, it already exists, and the value is being built within it.